modules/combined/test/tests/gap_heat_transfer_htonly/tests

```diff
@@ -30,15 +30,15 @@
   [../]
   [./ZR]
     type = Exodiff
-    input = 'gap_heat_transfer_htonly_zr_test.i'
+    input = 'gap_heat_transfer_htonly_rz_test.i'
     exodiff = 'gap_heat_transfer_htonly_zr_test_out.e'
     abs_zero = 1e-9
     rel_err = 5e-3
     compiler = 'CLANG GCC'
     requirement = "Thermal contact shall solve cylindrical heat transfer for a constant conductivity gap in 2D axisymmetric coordinates where the axial axis is along the x-direction"
     design = 'source/bcs/GapHeatTransfer.md'
     issues = '#12071'
```

Comment:
I am not sure how important it is to reference all issues here but you might want to add your current issue number as well for this test.